content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -383,6 +383,12 @@ Run 'kubectl get nodes' on control-plane to see this machine join.
 A few seconds later, you should notice this node in the output from `kubectl get
 nodes` when run on the control-plane node.
 
+{{< note >}}
+As the cluster nodes are usually initialized sequentially, the CoreDNS Pods are likely to 
+all run on the first control-plane node. If you install a cluster with more than one node, 
+we recommend that you rebalance the CoreDNS Pods with `kubectl -n kube-system rollout restart deployment coredns`.
```

Comment:
nit ([avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we), and timing for when to do this):
```suggestion
rebalance the CoreDNS Pods with `kubectl -n kube-system rollout restart deployment coredns`
after you first add nodes that are not part of the control plane.
```

How about that?

Reply:
Thanks @sftim 
Updated.